### ISSUE TITLE
False positive for info LC0091 (missing translation)

### ISSUE THREAD

**Issue by NKarolak** [StefanMaron/BusinessCentral.LinterCop#1004]
**Affecting preview version 0.35.1.0 only!**

In my workspace, I added a field to my setup table that had been fully translated until recently. Now, of course, it isn't anymore:

![Image](https://github.com/user-attachments/assets/b090f0f6-430c-469a-aa7f-02ebf12082b8)

However, the **first** entry is incorrect, as it affects code that I didn't change. The NAB AL Tools extension also proves that the translation still exists:

![Image](https://github.com/user-attachments/assets/839f8164-072d-42e5-a1d2-fd12351b79fe)

This is the entry from the corresponding de-DE translation file:
```
        <trans-unit id="Table 4210039887 - Property 2879900210" size-unit="char" translate="yes" xml:space="preserve">
          <source>PDM Setup</source><target>PDM-Einrichtung</target>
          <note from="Developer" annotates="general" priority="2"></note>
          <note from="Xliff Generator" annotates="general" priority="3">Table CCS PDM Setup - Property Caption</note>
        </trans-unit>
```
The translation file itself is okay. There are other entries looking very similar to the one above, including the dash in the `<target>`, but they are still recognized by LinterCop.

"Reload Window" did not help.

**Comment by ralf-escher** [comment]
This rule seems not be stable, also see #948.
I did put it on `"action": "None"` meanwhile.

**Comment by ChristianHovenbitzer** [comment]
@NKarolak let me see if I can replicate that, although multi language should work on basic tables

**Comment by guidorobben** [comment]
I'm v0.38.1 and this generates random missing entries every time I compile.

**Comment by guidorobben** [comment]
If you need more info, let me know. I have everything translated and I still have like 50 warnings of missing translations.

**Comment by Arthurvdv** [comment]
I believe this is now resolved in the Pre-release [v0.38.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.38.1) version of the LinterCop.

See more details on https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/1068

**Comment by guidorobben** [comment]
Downloaded the new version. Seem to work as expected. Will let you know at the end of the day.

**Comment by guidorobben** [comment]
Seems to work fine now.
